docs/src/utils/getAllPaths.ts

```diff
@@ -0,0 +1,48 @@
+import { globby } from 'globby';
+import { FRAMEWORKS } from '@/data/frameworks';
+import { getContentPaths } from '@/utils/getContentPaths';
+import { getPageFromSlug } from '@/utils/getPageFromSlug';
+import { getPagesManifest } from '@/utils/getPagesManifest';
+import { META_INFO } from '@/data/meta';
+
+export async function getAllPaths() {
```

Comment:
This file is from docs/scripts/generate-sitemap-robotstxt.ts. I'll make a followup PR to clean docs/scripts/generate-sitemap-robotstxt.ts up